server/src/internalClusterTest/java/org/opensearch/search/basic/TransportTwoNodesSearchIT.java

```diff
@@ -205,6 +206,7 @@ public void testDfsQueryThenFetch() throws Exception {
 
     public void testDfsQueryThenFetchWithSort() throws Exception {
         prepareData();
+        indexRandomForConcurrentSearch("test");
```

Comment:
can `indexRandomForConcurrentSearch("test");` be added as part of ` prepareData()`?

Reply:
I had done it that way before, but then I noticed there is one method testFailedSearchWithWrongQuery that uses prepareData and isn't listed in the test cases that needed the concurrent search fix, so I changed it to call indexRandomForConcurrentSearch after prepareData

Reply:
i dont see any harm that indexRandomForMultipleSlices would cause to `testFailedSearchWithWrongQuery` test. Lets add it to prepareData() itself.

Reply:
Moved the call into prepareData and squashed the commits

Reply:
Removed changes to TransportTwoNodesSearchIT until future fix is applied